src/vector_index.h

```diff
@@ -59,7 +59,6 @@ typedef struct VectorFilter {
   };
   VectorQueryType type;               // vector similarity query type
   VectorQueryParam *params;           // generic params for the query
```

Comment:
`params` are already defined in `RSQueryNode`
(common to all query nodes, not only for `QueryVectorNode`)
Seems redundant here.

Reply:
here they come in different role - these are params for the vector index itself, for the VecSim library to use. maybe I should give it another name